src/types.ts

```diff
@@ -68,6 +68,11 @@ export interface DiscordUser {
 	username: string;
 	public_flags: number;
 	id: Snowflake;
+	global_name: string;
+	/**
+	 * @deprecated Use global_name instead.
+	 */
+	display_name: string;
```

Comment:
The value should be checked since this feature is still rolling out for everyone yes?
![image](https://github.com/alii/use-lanyard/assets/77577746/fc0c3cc1-7725-4ca8-908c-81e2f425c55d)

```suggestion
	global_name: string | null;
	/**
	 * @deprecated Use global_name instead.
	 */
	display_name: string | null;
```